tests/components/samsungtv/test_config_flow.py

```diff
@@ -35,11 +35,9 @@
 from homeassistant.config_entries import ConfigEntryState
 from homeassistant.const import (
     CONF_HOST,
-    CONF_IP_ADDRESS,
     CONF_MAC,
     CONF_METHOD,
     CONF_MODEL,
-    CONF_NAME,
```

Comment:
I think this change doesn't belong to this PR

Reply:
Following #144372, it is now only used in `MOCK_IMPORT_DATA` and `MOCK_IMPORT_WSDATA` 
Since now I am removing these unused "import" constants, we don't need to import it anymore

Reply:
If you prefer, I can create a separate PR for removing the MOCK_IMPORT_*** constants

Reply:
> If you prefer, I can create a separate PR for removing the MOCK_IMPORT_*** constants

moved to #144394